src/Compatibility/Core/src/Android/Forms.cs

```diff
@@ -185,9 +186,12 @@ public static Color GetColorButtonNormal(Context context)
 			return _ColorButtonNormal;
 		}
 
+		public static void Init(IActivationState activationState) =>
+			Init(activationState.Context, activationState.SavedInstance);
+
 		// Provide backwards compat for Forms.Init and AndroidActivity
 		// Why is bundle a param if never used?
-		public static void Init(Context activity, Bundle bundle)
+		public static void Init(IMauiContext context, Bundle bundle)
```

Comment:
How would someone migrating their app use this? I would expect there might be some extra code to add, but maybe we can wrap these things for them automatically? Otherwise, people will not know what do do.

Reply:
Honestly I don't think we'll even have an Init class once we ship.... The reason we're calling init right now is because we haven't broken up Forms/Platform/Device classes

Once those are deconstructed into Maui concepts we won't need Init. What we will need is a way to opt into Assembly scanning for old school renderers if people still want to enable that. That's something we will have on the builder

builder.ScanForCompatibilityRenderers()

or something like that